libs/env/index.js

```diff
@@ -1,13 +1,19 @@
+const fs = require("fs");
```

Comment:
@fiji-flo Does it make sense to distinguish between ENV variables (`lib/env`) and constants (`lib/constants`), or should I just merge them into one?

Reply:
(I think it makes sense because `lib/constants` doesn't depend on any dotenv/fs/path import.)